src/consensus/consensus.rs

```diff
@@ -17,6 +18,8 @@ pub enum MalachiteEventShard {
 pub enum SystemMessage {
     MalachiteNetwork(MalachiteEventShard, MalachiteNetworkEvent), // Shard Id and the malachite network event
     Mempool(MempoolMessageWithSource),
+
+    DecidedValue(proto::DecidedValue),
```

Comment:
We should make this clearer that it's for a read node only, maybe something like `ReadNodeReceiveDecidedValue`